lego/apps/events/tests/test_penalties.py

```diff
@@ -53,16 +53,22 @@ def test_get_earliest_registration_time_one_penalty(self):
 
         user = get_dummy_users(1)[0]
         AbakusGroup.objects.get(name="Webkom").add_user(user)
-        Penalty.objects.create(user=user, reason="test", weight=1, source_event=event)
+        Penalty.objects.create(
+            user=user, reason="first test penalty", weight=1, source_event=event
+        )
+        Penalty.objects.create(
+            user=user, reason="second test penalty", weight=2, source_event=event
+        )
         penalties = user.number_of_penalties()
 
         earliest_reg = event.get_earliest_registration_time(
             user, [webkom_pool], penalties
         )
-        self.assertEqual(earliest_reg, current_time + timedelta(hours=3))
+        self.assertEqual(earliest_reg, current_time + timedelta(hours=5))
 
+    """
```

Comment:
I don't really see the point of commenting out these tests, why not delete them completely?

If we need to revert the code, we'll also revert the tests - plus it makes it simpler to review if the tests are really ready to be deleted if the entire test is deleted and thus visible in the diff

Reply:
That is a really good point. Thank you. I will delete them immediately